pkg/kn/commands/source/list_types.go

```diff
@@ -47,13 +52,17 @@ func NewListTypesCommand(p *commands.KnParams) *cobra.Command {
 			}
 
 			sourceListTypes, err := dynamicClient.ListSourcesTypes()
-			if err != nil {
-				return err
+			switch {
+			case knerrors.IsForbiddenError(err):
```

Comment:
I can comment lines 56-59 and still pass all tests

Reply:
added https://github.com/knative/client/issues/952 to track adding mock client for dynamic client, with which we should be able to pass the errors to expect and verify this in unit tests, can be added in a separate PR, sounds good?